test/hermes/intl/test262-to-locale-lowercase.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
```

Comment:
Since this is adapted from test262, we should reproduce their copyright notice here.

```suggestion
/* This code is adapted from Test262, which has the following copyright notice:
*
* The << Software identified by reference to the Ecma Standard* ("Software)">>  is protected by copyright and is being 
* made available under the  "BSD License", included below. This Software may be subject to third party rights (rights 
* from parties other than Ecma International), including patent rights, and no licenses under such third party rights 
* are granted under this license even if the third party concerned is a member of Ecma International.  SEE THE ECMA 
* CODE OF CONDUCT IN PATENT MATTERS AVAILABLE AT http://www.ecma-international.org/memento/codeofconduct.htm FOR 
*INFORMATION REGARDING THE LICENSING OF PATENT CLAIMS THAT ARE REQUIRED TO IMPLEMENT ECMA INTERNATIONAL STANDARDS*.
*
* Copyright (C) 2012-2013 Ecma International
* All rights reserved.
*/

```